payments/payment_manager_test.go

```diff
@@ -0,0 +1,130 @@
+package payments
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/statechannels/go-nitro/internal/testactors"
+	. "github.com/statechannels/go-nitro/internal/testhelpers"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// manager lets us implement a getBalancer helper to make test assertions a little neater
+type manager interface {
+	Balance(chanId types.Destination) (Balance, error)
+}
```

Comment:
Could be worth extracting this into a (non-test) interface (see other comment).

Reply:
I'm not sure how useful that would be; if it becomes useful, could we export it as a public interface then?

I understood the (a?) rule of thumb in go is to define interfaces where types are _consumed_, not where types are _defined_. So, someone who wants to use either a PaymentManager or a ReceiptManager, just to call the `Balance` function, would define a `Manager` interface.